src/core/ext/filters/stateful_session/stateful_session_filter.cc

```diff
@@ -147,14 +146,14 @@ absl::string_view GetClusterToUse(
       service_config_call_data->GetCallAttribute<XdsClusterAttribute>();
   GPR_ASSERT(cluster_attribute != nullptr);
   auto cluster_to_use = cluster_attribute->cluster();
```

Comment:
Since you've changed the logic here to no longer actually use this variable as the cluster to use in all cases, I suggest renaming this to `current_cluster`.

Reply:
Done